packages/react-components/react-portal/src/components/Portal/usePortalMountNode.ts

```diff
@@ -57,7 +57,7 @@ export const usePortalMountNode = (options: UsePortalMountNodeOptions): HTMLElem
     }
   }, [element, className, dir]);
 
-  (useKeyboardNavAttribute() as React.MutableRefObject<HTMLElement>).current = element!;
+  (useFocusVisible<HTMLDivElement>() as React.MutableRefObject<HTMLElement | null>).current = element;
```

Comment:
I haven't tested it, but it seems that the ref should be assigned in `useEffect()` to work properly in Strict mode.